apps/server/src/services/ElasticSearchClient.test.ts

```diff
@@ -268,44 +268,51 @@ describe("indexElasticSearch", () => {
     expect(institutionsIndexedCount).toEqual(3);
   });
 
-  it("it indexes institutions retrieved from the institution server", async () => {
-    ElasticSearchMock.clearAll();
-
-    server.use(
-      http.get(config.INSTITUTION_CACHE_LIST_URL, () => {
-        return HttpResponse.json([
-          elasticSearchInstitutionData,
-          elasticSearchInstitutionData,
-        ]);
-      }),
-    );
+  describe.each([
+    { singleThread: true, description: "with single-threading" },
+    { singleThread: false, description: "without single-threading" },
+  ])("Index Elasticsearch $description", ({ singleThread }) => {
+    it("indexes institutions retrieved from the institution server", async () => {
+      config.ELASTIC_SEARCH_SINGLE_THREAD = singleThread;
```

Comment:
I would mock the config instead of modifying it directly. This feels like it could leak into other tests. We use getConfig in a few other places to make it mockable